acceptance/tests/facter_flush.rb

```diff
@@ -0,0 +1,52 @@
+test_name 'facter should flush fact values' do
+  tag 'risk:high'
+
+  fact_content1 = <<-EOM
+    require 'facter'
+
+    Facter.add(:fact1) do
+      'this should be flushed'
+    end
+
+    Facter.flush
+
+    puts "Fact1: \#\{Facter.value(:fact)\}"
```

Comment:
you are asking for a fact that doesn't exist

Reply:
Yes, missed that. Thanks